testnet/launcher/docker.go

```diff
@@ -219,10 +219,9 @@ func waitForHealthyNode(port int) error { // todo: hook the cfg
 	reqBody := `{"method": "obscuro_health", "id": 1}`
 
 	timeStart := time.Now()
-	defer func() { fmt.Printf("Node became healthy after %f seconds\n", time.Since(timeStart).Seconds()) }()
```

Comment:
Moved this to just log on success after the retry because was getting confused by log saying it was healthy when it actually failed with timeout.